airflow/models/pydantic/dagrun.py

```diff
@@ -14,35 +14,32 @@
 # KIND, either express or implied.  See the License for the
```

Comment:
I renamed this one to make sense with dagrun.py in `airflow/models`. Happy to revert if it does not make sense